src/views/notification/View.vue

```diff
@@ -0,0 +1,133 @@
+                          <div class="truncate sm:whitespace-normal flex items-center">
+                            <MailIcon class="w-4 h-4 mr-2"/>{{ this?.notification?.user?.email }}
+                          </div>
+                          <div class="truncate sm:whitespace-normal flex items-center mt-3">
+                            <UserIcon class="w-4 h-4 mr-2"/>{{ this?.notification?.user?.pre_name }} {{ this?.notification?.user?.last_name }}
+                          </div>
+                          <div class="truncate sm:whitespace-normal flex items-center mt-3">
+                            <HashIcon class="w-4 h-4 mr-2"/>User ID: {{ this?.notification?.user?.id }}
+                          </div>
+                        </div>
+                      </div>
+                    </div>
+                  </div>
+                  <div class="p-5">
+                    <div class="flex flex-col-reverse xl:flex-row flex-col">
+                      <div class="flex-1 mt-6 xl:mt-0">
+                        <div class="grid grid-cols-12 gap-x-5 mb-3">
+                          <div class="col-span-12 xxl:col-span-12 mb-4">
+                            <div>
+                              <label for="update-title-notification" class="form-label">
+                                Title
+                              </label>
+                              <input
+                                id="update-title-notification"
+                                type="text"
+                                :class="'form-control' + (this.validation_error?.title != null ? ' border-theme-6' : '')"
+                                placeholder="Enter Title"
+                                v-model="this.notification.title"
+                                disabled
+                              />
+                              <div v-if="this.validation_error?.title != null" class="text-theme-6 mt-2 mb-4">
+                                {{ this.validation_error?.title[0] }}
+                              </div>
+                            </div>
+                          </div>
+                          <div class="col-span-12 xxl:col-span-12 mb-4">
+                            <div>
+                              <label for="update-title-notification" class="form-label">
+                                Content
+                              </label>
+                              <textarea
+                                id="update-content-notification"
+                                :class="'form-control' + (this.validation_error?.content != null ? ' border-theme-6' : '')"
+                                placeholder="Enter Content"
+                                rows="10"
+                                v-model="this.notification.content"
+                                disabled
+                              />
+                              <div v-if="this.validation_error?.content != null" class="text-theme-6 mt-2 mb-4">
```

Comment:
`!= null` is unnecessary, questionmark-syntax is taking a valid value as true by default

Reply:
Solved in https://github.com/KrbDevelopment/Articly-Frontend/commit/b368ca9a2e2d4ff88e6b389b1517fc6ec13ecf4b